custom_components/electrolux_status/button.py

```diff
@@ -84,7 +86,7 @@ def entity_domain(self):
     @property
     def unique_id(self) -> str:
         """Return a unique ID to use for this entity."""
-        return f"{self.config_entry.entry_id}-{self.val_to_send}-{self.entity_attr}-{self.entity_source}-{self.pnc_id}"
+        return f"{self.config_entry.entry_id}-{self.val_to_send}-{self.entity_name}-{self.entity_source}-{self.pnc_id}"
```

Comment:
I've set `entity_name` here to conform to your last commit - but giving it some more thought `entity_attr` is probably better as it will be unique. The truncated version could cause duplicate id's.

I suspect the 2.0.5 change to use truncated might have actually caused some more `_2` cases too